fala/templates/adviser/category_search.html

```diff
@@ -66,6 +66,9 @@ <h1 class="govuk-heading-xl"> {{_('Find a legal aid adviser for ')}}
     <div class="govuk-grid-column-two-thirds">
       <form action="{{ search_url }}" method="get" id="fala_questions_for_tailored_results">
         <input type="hidden" name="categories" value="{{ category_code }}">
+        {% if sub_category_code %}
```

Comment:
should this `name` be different so we can differentiate it from the element above?

Reply:
@willc-work this name has to stay the same. `categories` is the dat input name and the key to the query param. It help populates the form to conduct the search.